docs/configuration.md

```diff
@@ -30,6 +30,7 @@ Some less common environment variables are:
 | `LOG_LEVEL_IN_STRING`         | By default, when using the `json` format, the level printed in the log records is an int (`10`, `20`, ..). This option tells the logger to print level as a string: `{"level": "info"}`. Default `false`                                                                                                                                            |
 | `SENTRY_DSN`                  | Set to a [Sentry](https://sentry.io/) DSN to report all errors thrown by your app. <p>_(Example: `https://1234abcd@sentry.io/12345`)_</p>                                                                                                                                                                                                           |
 | `PORT`                        | The port to start the local server on. Default: `3000`                                                                                                                                                                                                                                                                                              |
+| `BASE_ADDRESS`                | The IP address/hostname to start the local server on. Default: `localhost`                                                                                                                                                                                                                                                                          |
```

Comment:
I think that should have been `BIND_ADDRESS`?
```suggestion
| `BIND_ADDRESS`                | The IP address/hostname to start the local server on. Default: `localhost`                                                                                                                                                                                                                                                                          |
```

Reply:
Good catch!